incubator/hnc/api/v1alpha1/hnc_config.go

```diff
@@ -156,8 +155,7 @@ type HNCConfigurationCondition struct {
 	//
 	// Currently, the supported values are:
 	//
-	// - "critSingletonNameInvalid": the specified singleton name is invalid. The name should be the
-	// same as HNCConfigSingleton.
+	// (We don't have any critical conditions for now.)
```

Comment:
We don't need this comment IMO, also it would look weird as the beginning of a list. Maybe just delete the paragraph on critical conditions completely?

Reply:
ah sg!